### DISCUSSION TITLE
Is it possible to edit a Blu-Ray PGS.sup dimensions without converting it to SRT/ASS?

### DISCUSSION THREAD

**Discussion by 0TTA** [SubtitleEdit/subtitleedit#6594] [Q&A] [answered]
Is it possible to edit a Blu-Ray PGS.sup dimensions without converting it to SRT/ASS?

**Comment by niksedk** [comment, marked as answer]
In File - Import - Blu-ray sup subtitle file for edit...

Change the video res (+ fix some alignment, use list view context menu):

![image](https://user-images.githubusercontent.com/1261331/212544096-ae21d2df-bf1a-4f0d-859a-d75950dc9a0a.png)

**Reply by 0TTA** [reply]
Not all heroes wear capes, thank you!
